pom.xml

```diff
@@ -143,7 +143,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.21</version>
+            <!--<version>2.23</version> -->
+            <version>2.25-rc775.35b435d87ee7</version>
```

Comment:
Only until I cut the release, then it will be converted to the final release version and this will be cleaned up